### DISCUSSION TITLE
[Feature Request] SearchMultipleInput

### DISCUSSION THREAD

**Discussion by kahirokunn** [heroui-inc/heroui#1512] [Feature requests]
### Is your feature request related to a problem? Please describe.

I prefer to use MultipleInput, but if the number of items is too large, you will want to narrow it down with text input.

### Describe the solution you'd like

This is exactly the kind of thing I want. But it is multiple choice!
https://mui.com/material-ui/react-autocomplete/#search-input

### Describe alternatives you've considered

Other experiences may be acceptable if multiple selections and refinement can be made.

**Comment by MeonisRP** [comment]
I would LOVE to see this component in NextUI. In my current project I use this functionality everywhere

**Comment by marlingDEV** [comment]
+1

**Comment by wingkwong** [comment]
let's track here - #4684
